_health-care/_js/components/form-elements/GrowableTable.jsx

```diff
@@ -28,37 +46,77 @@ class GrowableTable extends React.Component {
     this.props.onRowsUpdate(rows);
   }
 
+  // TODO: change view to not be dependent on index number, this is creating UI problems
   render() {
+    let reactKey = 0;
```

Comment:
@awong-dev I'm a little confused, how can we store them outside of render if render is called whenever there is a state change, which might affect how we render the rowElements?